hub/core/chunk_engine.py

```diff
@@ -463,28 +567,26 @@ def update(
         global_sample_indices = tuple(index.values[0].indices(self.num_samples))
         for i, sample in enumerate(samples):
             global_sample_index = global_sample_indices[i]  # TODO!
-            chunks = self.get_chunks_for_sample(global_sample_index, copy=True)
-            if len(chunks) > 1:
-                raise NotImplementedError(
-                    "You can't update a sample that is present in multiple chunks."
+            if self._is_tiled_sample(global_sample_index):
+                self._update_tiled_sample(global_sample_index, index, sample)
```

Comment:
is tensor meta shape interval being updated here?

Reply:
Yes.. updated.